platform/ui/src/components/Select/Select.jsx

```diff
@@ -1,61 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
-import ReactSelect from 'react-select';
+import ReactSelect, { components } from 'react-select';
 
 import './Select.css';
 
+const MultiValueContainer = ({ data }) => {
+  return <span>{`${data.label}, `}</span>;
+};
+
+const MultiValue = props => {
+  return (
+    <components.MultiValue {...props}>
+      <span>{props.data.label}</span>
+    </components.MultiValue>
+  );
+};
+
+const Option = props => {
+  return (
+    <div>
+      <components.Option {...props}>
+        <input
+          type="checkbox"
+          checked={props.isSelected}
+          className="w-6 h-6"
+          onChange={e => null}
+        />{' '}
```

Comment:
Was this "blank space" `{' '}` intentional?

Reply:
Checkbox + Text is not aligned:

<img src="https://user-images.githubusercontent.com/1905961/77666804-29f9c600-6f60-11ea-83b2-5bdae0cef74a.png" width="300" />

Reply:
I think autofix inserts the blank space to preserve line breaks that would not persist with current formatting rules.

Reply:
Fixed alingment and also the extra `{' '}`.